python/mspasspy/db/database.py

```diff
@@ -54,7 +68,7 @@ def read_distributed_data(
     normalize=None,
     load_history=False,
     exclude_keys=None,
-    format="dask",
+    format=None,
```

Comment:
Change the default to none doesn't make sense here. The default should be dask or pyspark if only one of the two exist, and it should be dask if both exist. Then, if neither of them exists, this function shouldn't be imported at all.